Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ruby
+
+RUN gem install --no-document etcenv
```

Comment:
Right, building from source is even better ;) 

But bundler is not needed here. I changed it to use the normal gem build procedure. Just take whichever you like best..